dist/amazon-linux/2/after-remove.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -x
+
+SERVICE="metrist-orchestrator.service"
+
+systemctl stop $SERVICE || true
+systemctl reset-failed $SERVICE || true
```

Comment:
I've had to do this because yum removes the unit files but doesnt stop the service. But when a service is stopped because it failed, it'll get marked as failed and will show up when you run `systemctl status metrist-orchestrator.service`